hwilib/devices/trezor.py

```diff
@@ -220,7 +224,7 @@ def ignore_input():
                 if our_keys > passes:
                     passes = our_keys
 
-                if not found:
+                if not found and not found_in_sigs:
```

Comment:
can you add some more comments/explanation in the code for this logic? I don't think this is obvious.

Reply:
Added some comments.